.github/workflows/release-build.yml

```diff
@@ -21,7 +21,20 @@ permissions:
   contents: write
 
 jobs:
+  wait-for-main-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for E2E test to succeed
+        uses: ezhang6811/wait-on-check-action@6275ea93847ce5952ab8ceeac8fb1111820cf44d
```

Comment:
Avoid using personal accounts for production release workflows. You could move the logic from the custom action to here if possible?

Reply:
fixed, replaced with local script